capten/config-worker/Dockerfile

```diff
@@ -6,7 +6,6 @@ RUN go mod download
 RUN CGO_ENABLED=0 go build -o ./build/config_worker config-worker/main.go
 
 FROM scratch
-COPY --from=builder ./build/config_worker config_worker
-
-USER 65532:65532
-ENTRYPOINT ["./config_worker"]
+COPY --from=builder ./build/config_worker /config_worker
+COPY --from=builder ./config-worker/git-config-files /git-config-files
```

Comment:
rename folder to /gitopts-template

under this folder different usecase to be used.

 /gitopts-templates
   /cicd
   /scm

for tekton use case, folder structure needed to be added under /cicd folder, not single file.